universal-application-tool-0.0.1/app/forms/MultiOptionQuestionForm.java

```diff
@@ -143,27 +146,42 @@ public QuestionDefinitionBuilder getBuilder() {
       predicateBuilder.setMaxChoicesAllowed(getMaxChoicesAllowed());
     }
 
-    ImmutableList.Builder<QuestionOption> questionOptions = ImmutableList.builder();
+    ImmutableList.Builder<QuestionOption> questionOptionsBuilder = ImmutableList.builder();
     Preconditions.checkState(
         this.optionIds.size() == this.options.size(),
         "Option ids and options are not the same size.");
 
+    if (this.optionIds.size() > 0) {
```

Comment:
This presetting to "something" isn't needed anymore right?

Reply:
Removed